bc_obps/reporting/tests/service/test_report_activity_save_service/test_save_report_methodology.py

```diff
@@ -0,0 +1,95 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from reporting.models.methodology import Methodology
+from reporting.tests.utils.report_data_bakers import report_emission_baker
+from reporting.service.report_activity_save_service import ReportActivitySaveService
+from reporting.tests.service.test_report_activity_save_service.infrastructure import TestInfrastructure
+from model_bakery.baker import make_recipe
+
+
+class TestSaveReportMethodology(TestCase):
+    def test_save_methodology(self):
+        test_infrastructure = TestInfrastructure.build()
+        report_activity = test_infrastructure.make_report_activity()
+        make_recipe("reporting.tests.utils.gas_type", chemical_formula="GGIRCA")
+        report_emission = report_emission_baker()
+
+        service_under_test = ReportActivitySaveService(
+            report_activity.report_version.id,
+            report_activity.facility_report.facility.id,
+            report_activity.activity.id,
+            test_infrastructure.user.user_guid,
+        )
+
+        # test for no report emission, should not be allowed due to model constraints
+        with self.assertRaises(ValidationError):
+            service_under_test.save_methodology(None, {"methodology": "Default HHV/Default EF"})
+
```

Comment:
We're basically testing django with this statement, which is already tested (or we hope so, but it's not our responsibility to make sure it works as intended)